packages/format-message-interpret/index.js

```diff
@@ -138,6 +138,21 @@ function interpretSelect (locale, id, children) {
 
 function interpretSimple (id) {
   return function format (args) {
-    return '' + args[id]
+    return '' + getArg(id, args)
+  }
+}
+
+function getArg (id, args) {
+  if (args[id] !== undefined) return args[id]
```

Comment:
Yeah we can use `if (id in args)` and yes, that's a good question. I would say in the example if `'x.y'` is set to anything it'll be used, so `null` will be returned. I'd avoid checking for falsy values to keep things simple and I think we could maybe assume that users of the library will be either using periods in var names or nested objects but unlikely a mixture of both. What are your thoughts? I guess the alternative would be to check for `undefined` or `null`. Happy to go for whichever you prefer here. I'd be leaning towards using `if (id in args)` though and can update that if you agree.